prbt_hardware_support/spec.dox

```diff
@@ -154,4 +154,18 @@
  *    - In T1 the speed override is 0.1.
  *    - In AUTOMATIC the speed override is 1.0.
  *    - If the operation mode is unknown, the speed override is set to zero.
+ *
+ *
+ * @subsection no_safety Other Setups (No ISO-10218-1 Support)
+ *
+ * @subsubsection simulated_robot Simulated Robot
+ *
+ * @definereq{speed_override_for_simulated_robot}
+ * - If the system is connected to a simulated robot instead of a real robot, the speed override (\ref speed_override) is 1.0.
```

Comment:
For now the formulation is ok because it simply reflects the state of our system. However, I strongly suggest that we introduce a mechanism to simulate the operation mode in simulation mode. Maybe we should open an issue for this?

Reply:
We should also discuss which parameter options (brake_test support, operation mode support, etc.) we want to support in simulation mode. -> Meeting?
I suggest to support only two variants:
- ISO `on`
- ISO `off`

Reply:
Thanks for having a look. The meeting already took place. We decided against "simulated ISO".